ush/ufoeval/test_yamls.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# test_yamls.sh 
+# This scripts runs run_ufo_hofx_test.sh for the input YAMLs in ../../parm/atm/obs/testing
+# and outputs pass/fail information
+
+# Work directory
+WORKDIR=$STMP/test_yamls
```

Comment:
what if the user does not have `$STMP` defined?

Reply:
The intention was for this line to be edited.  What would you suggest?

Reply:
This is fine, just was being thorough with the review.